extensions/partitioning.md

```diff
@@ -0,0 +1,35 @@
+# Partitioning extension
+
+This extension defines attributes for use by message brokers and their
+clients that support partitioning of events, typically for the purpose of
+scaling.
+
+## Attributes
+
+### partitionKey
```

Comment:
We're lower-casing all of our attribute names now, so `s/K/k/`

Reply:
Updated.

Reply:
thanks!